circle.yml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  override:
+    - printf '%s\n' color=false progress=false >~/.npmrc
+    - make setup
+
+machine:
+  node:
+    version: 4
```

Comment:
Fair enough. My preference is to always put code into production with provisioning scripts. Retrofitting is always harder.